deploy/helm/node-feature-discovery-operator/templates/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |+
+    apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+    kind: ControllerManagerConfig
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: 39f5e5c3.nodefeaturediscoveries.nfd.kubernetes.io
+
+kind: ConfigMap
+metadata:
+  labels:
+    {{- include "node-feature-discovery-operator.labels" . | nindent 4 }}
+  name: {{ include "node-feature-discovery-operator.fullname" . }}-manager-config
+  namespace: {{ include "node-feature-discovery-operator.namespace" . }}
```

Comment:
nit: for readability (and consistenc) I'd move these fields to the beginning of the yaml, just like for other objects